runtime/runtime_test.go

```diff
@@ -5282,12 +5285,24 @@ func TestRuntimeComputationLimit(t *testing.T) {
 
 			storage := newTestLedger(nil, nil)
 
+			var computationUsed uint
+
+			checkComputationLimit := func(_ MetringOperationType, intensity uint) error {
+				computationUsed += intensity
+				if computationUsed <= computationLimit {
+					return nil
+				}
+				panic(ComputationLimitExceededError{
+					Limit: computationLimit,
+				})
+			}
```

Comment:
This should be moved to the FVM now, correct?

Reply:
yeah, just kept it here for testing that there is no side effect due to the change until we add proper tests on the FVM side, then we could remove these